junction/static/less/variables.less

```diff
@@ -0,0 +1,10 @@
+/*
+  Color pallette can be found here
+  http://coolors.co/fa8072-4e4e4e-fff7f8-555358-5f6062
+*/
+
+@junction-color-1: darken(#FA8072, 10%); //salmon
+@junction-color-2: #4e4e4e; //dark gray
+@junction-color-3: #FFF7F8; //variant of white
+@junction-color-4: #555358; //gray-darker
+@junction-color-5: #5F6062; // gray
```

Comment:
Actually I did this deliberately so that we can have more flexibility to change the color scheme, say if you want your primary color to be green then first you might want to rename `junction-blue`  to `junction-green` & replace all the occurrences. This way you just have to make change in one place, thoughts ? But again if we are not going to change color scheme often then we can use the descriptive names